src/Blocks/components/submit/components/submit-editor.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classnames from 'classnames';
+import { selector, checkAttr, props } from '@eightshift/frontend-libs/scripts';
+import { FieldEditor } from '../../../components/field/components/field-editor';
+import manifest from '../manifest.json';
+
+export const SubmitEditor = (attributes) => {
+	const {
+		componentClass,
+	} = manifest;
+
+	const {
+		selectorClass = componentClass,
+		blockClass,
+		additionalClass,
+	} = attributes;
+
+	const submitValue = checkAttr('submitValue', attributes, manifest);
+	const submitType = checkAttr('submitType', attributes, manifest);
+
+	const submitClass = classnames([
+		selector(componentClass, componentClass),
+		selector(blockClass, blockClass, selectorClass),
+		selector(additionalClass, additionalClass),
+	]);
+
+	const submit = (
+		<input
+			type='submit'
+			value={submitValue}
+			className={submitClass}
+		/>
+	);
```

Comment:
What is the use case for using `input` instead of `button type="submit"`?

Reply:
Not sure but I added the option if there is any need for this for any reason